docs/packages/Package Development/package development main.rst

```diff
@@ -3,20 +3,18 @@
 Package Development: A "How-to Guide" for Developing Packages for Others to Use
 ================================================================================
 
-The purpose of this page is to get your mesa package sharing as quickly as possible. The authoritative guide for python package development is through the `Python 
-Packaging User Guide <https://packaging.python.org/>`_. This will take you through the entire process necessary for getting you package on the Python Package Index.
+The purpose of this page is to get your mesa package sharing as quickly as possible. 
```

Comment:
Change to: The purpose of this page is help you set up and distribute your mesa package as quickly as possible.